org.kde.kamoso.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "org.kde.kamoso",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kamoso",
+    "rename-icon": "kamoso",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=host",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11",
```

Comment:
```suggestion
        "--socket=wayland",
        "--socket=fallback-x11",
```

Reply:
KDE Apps still use x11 here for now. I don't remember the details.

Reply:
Well, I see two possible reasons:
1. they were pacakged before the fallback did exist
2. they set the QPA do 'xcb' because of Qt bugs with wayland and didn't remove the wayland socket access.

Reply:
I tried using fallback-x11 and it failed on QPA trying to do 'xcb' so that's most probably it. @tsdgeos Do you remember what we would need to move to wayland by default and X11 as fallback here?

Reply:
then you remove the wayland socket.

Reply:
I don't see were the app set the QPA to `xcb` though.